Exiled.API/Features/Player.cs

```diff
@@ -1528,6 +1528,21 @@ public byte GetEffectIntensity<T>()
             throw new ArgumentException("The given type is invalid.");
         }
 
+        /// <summary>
+        /// Gets all the active <see cref="PlayerEffect"/>.
+        /// </summary>
+        public ReadOnlyCollection<PlayerEffect> GetAllEffects()
+        {
+            ReadOnlyCollection<PlayerEffect> ActiveEffects = new ReadOnlyCollection<PlayerEffect>();
+            for (int i = 0; i < Enum.GetValue(typeof(EffectType)).GetUpperBound(0); i++)
+            {
+              if (GetEffect((EffectType) i).Intensity > 0)
+                ActiveEffects.Add(GetEffect((EffectType)i));
+            }
+            
+            return ActiveEffects;
```

Comment:
Why not simply use `ReferenceHub.playerEffectsController.AllEffects`? I don't like the idea of getting the enum upper bounds constantly, then getting every effect twice.